core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1394,7 +1394,7 @@ paths:
         500:
           $ref: '#/components/responses/RawlsInternalError'
       x-codegen-request-body-name: newEntityNameJson
-  /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/renameEntityType:
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/renameEntityType:
```

Comment:
if we're gonna move this one, how about moving it to
```
/api/workspaces/{workspaceNamespace}/{workspaceName}/entityTypes/{entityType}/renameEntityType
```
to make its URL exactly parallel to `renameAttribute`? I know that would require changing the payload of that other API as well.